types/tests/banana-test.ts

```diff
@@ -0,0 +1,33 @@
+import Banana from '../index';
```

Comment:
Can this be written as actual unit test? We use mocha in banana-i18n

Reply:
Not really. [Unfortunately] I followed here the pattern of vuejs. See: https://github.com/vuejs/vue/tree/dev/types.

Reply:
We can use their style of testing. See https://github.com/vuejs/vue/blob/dev/package.json#L36

Reply:
Done

Reply:
Isn't it writing a mocha based test for a file with assertions a normal practice? What exactly the problem in writing a mocha test(see a sample tutorial https://journal.artfuldev.com/write-tests-for-typescript-projects-with-mocha-and-chai-in-typescript-86e053bdb2b6)?

Reply:
I do not see any problem to write test, but here are my but's:
a) Iff I understand the purpose of test correctly (independent of the framework) test should cover the behavior of modules and should not care about implementation detail (like types). 
b) Firstly, thanks for the article. As the article also notice it's about BDD, which you already (thankfully) have done. So here my point, since you already test the behavior of the module what should I test? Should I take the already made up tests and throw TypeScript? Or should I test that part "I have under control"? Of course you can "testing the types decoupled of the implementation smells" - there you would be right, totally. Which brings us back to should we paint all the tests, with TypeScript, since these are places where BDD and magic is happening. Anyhow, if you agree to this, this should be then rolled out in 2nd commit, I guess.